src/cmd_line/commands/vscode.ts

```diff
@@ -0,0 +1,27 @@
+import * as vscode from 'vscode';
+import { VimState } from '../../state/vimState';
+import { CommandBase } from '../node';
+import { Scanner } from '../scanner';
+
+export class VsCodeCommand extends CommandBase {
+  private command?: string;
+
+  constructor(command?: string) {
+    super();
+    this.command = command;
+  }
+
+  public static parse(args: string): VsCodeCommand {
+    const scanner = new Scanner(args);
+    scanner.skipWhiteSpace();
+    return new VsCodeCommand(scanner.isAtEof ? undefined : scanner.nextWord());
+  }
+
+  private async vsc() {
+    await vscode.commands.executeCommand(this.command ?? '');
```

Comment:
What does `executeCommand('')` do? I'd think `:vsc` with no argument should just throw `E471`